### PR TITLE
Fix for global configuration overrides (Issue 449)

### PR DESCRIPTION
**Pull request by shubDhond**
### Description of change

Fixed ChartJsProvider.setOptions to reflect changes to options object to Chart.defaults options so that options can actually be applied to the Charts.

### Pull Request check-list

- [x] Run `gulp test` to ensure there are no linting, or style issues and all tests pass.
- [x] Squash your commits into a few commits only.
- [x] Make sure the commit message is short, concise and descriptive of the issues you're fixing.
- [x] Avoid mixing up multiple issues and/or features, open one pull request for each issue.
- [x] Have you updated the documentation and / or examples?
- [x] Have you included a new test?

### CODE REVIEW

File: angular-chart.js [hunk 1]
```diff
@@ -78,11 +78,13 @@
       // If no type was specified set option for the global object
       if (! customOptions) {
         customOptions = type;
-        options = angular.extend(options, customOptions);
-        return;
+        options = angular.merge(options, customOptions);
+      }else{
```
Comment (jtblin) [inline angular-chart.js]: Nit: spaces after } and before {

**Comment by jtblin** [comment]
Thanks a ton for the PR @shubDhond! Any chance you add unit / integration test for this?

**Comment by shubDhond** [comment]
Will do.

**Comment by jtblin** [comment]
Thanks hips! 

> ... added unit and integ tests. However I could not get any of the integ tests to pass including the ones which were already there.

Yeah the tests need to be run in docker via `npm run docker`. All good anyway, can see the checks are all green in Travis :)

**Comment by shubDhond** [comment]
When can I expect this to be published? Sort of need this for a project.

**Pull request merged**